build/Targets/Settings.targets

```diff
@@ -128,15 +128,16 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
-  
+
   <!-- This is a really hacky way to detect whether we are on a legacy or a willow based VS install.
        Basically, we check for a registry key that will only exist in legacy VS installs, and assume
        we are a willow based installation if our VSVersion is 15.0 and the registry key doesn't exist. -->
   <PropertyGroup>
     <IsVSBeforeDev15Preview4 Condition="'$(OS)' == 'Windows_NT' AND '$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\15.0@InstallDir)' != ''">true</IsVSBeforeDev15Preview4>
     <IsVSBeforeDev15Preview4 Condition="'$(OS)' == 'Windows_NT' AND '$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\VisualStudio\15.0@InstallDir)' != ''">true</IsVSBeforeDev15Preview4>
     <IsVSBeforeDev15Preview4 Condition="'$(IsVSBeforeDev15Preview4)' == 'true' AND Exists('$(MSBuildBinPath)\..\..\..\Common7\IDE\devenv.isolation.ini')">false</IsVSBeforeDev15Preview4>
-    <IsVSBeforeDev15Preview4 Condition="'$(IsVSBeforeDev15Preview4)' == 'true' AND Exists('$(MSBuildBinPath)\..\..\..\..\Common7\IDE\devenv.isolation.ini')">false</IsVSBeforeDev15Preview4>
+    <IsDev15RC Condition="Exists('$(MSBuildBinPath)\..\..\..\..\Microsoft.VisualStudio.ExtensionEngine.dll')">true</IsDev15RC>
+    <IsDev15RC Condition="Exists('$(MSBuildBinPath)\..\..\..\..\Microsoft.VisualStudio.ExtensionEngine.dll')">true</IsDev15RC> <IsVSBeforeDev15Preview4 Condition="'$(IsVSBeforeDev15Preview4)' == 'true' AND Exists('$(MSBuildBinPath)\..\..\..\..\Common7\IDE\devenv.isolation.ini')">false</IsVSBeforeDev15Preview4>
```

Comment:
Looks like a copy/paste error here.  The `IsDev15RC` property is defined twice and another property is just off screen.